### PR TITLE
GPII-3467: Allow coverage sender to be used without QUnit or Testem.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [GPII-3467](https://issues.gpii.net/browse/GPII-3467) for details.

### CODE REVIEW

File: src/js/client/coverageSender.js [hunk 1]
```diff
@@ -80,4 +85,4 @@
             QUnit.done(afterTestsCallback);
         }
     };
-})(typeof Testem !== "undefined" ? Testem : false, QUnit);
+})(typeof Testem !== "undefined" ? Testem : false, typeof QUnit !== "undefined" ? QUnit : false);
```
Comment (amb26) [inline src/js/client/coverageSender.js]: Perhaps just window.Testem and window.QUnit is a touch more succinct?

File: docs/coverage.md [hunk 2]
```diff
@@ -39,3 +39,19 @@ Please note, as shown in the above example:
...
+
+## Coverage Sender Advanced Options
+
+The middleware that serves up the coverage sender accepts three options, which are used to generate the client-side code
+that your browser loads.
+
+| Option           | Type        | Description                           |
+| ---------------- | ----------- | ------------------------------------- |
+| `exposeCallback` | `{Boolean}` | Whether to expose the callback that sends coverage data as the global named variable `window.gpii.testem.coverage.afterTestsCallback` so that it can be called directly in situations where Testem and QUnit are not available. |
+| `hookTestem`     | `{Boolean}` | Whether to hook the callback that sends coverage data into the Testem lifecycle, so that test results are send when Testem detects that the tests on a page have completed.  Defaults to `true`. |
```
Comment (amb26) [inline docs/coverage.md]: typo this and next, send -> sent

File: src/js/client/coverageSender.js (hunk 1 above)
Comment (the-t-in-rtf) [reply src/js/client/coverageSender.js]: Agreed, and fixed.

File: docs/coverage.md (hunk 2 above)
Comment (the-t-in-rtf) [reply docs/coverage.md]: Fixed.

**Comment by the-t-in-rtf** [comment]
@amb26, I have addressed your feedback, this should be ready for further review.

**Pull request merged**